dtos/game/player.dto.ts

```diff
@@ -0,0 +1,4 @@
+export interface PlayerDto {
+  x: number;
```

Comment:
I am not sure that communicating the `x` coordinate to the server is useful.

Reply:
Actually, I'm going to suggest that we leave this for now and fix it later if we need to optimize. This way the position of the paddles is controlled by the backend and we don't have to worry about syncing the front and the back if we decide to change the starting position or allow game modes where it changes.

It shouldn't be too hard to fix, but it also is low priority